zebra-chain/src/orchard/tests/vectors/sinsemilla.rs

```diff
@@ -7,7 +7,7 @@ pub struct TestVector {
     pub(crate) hash: [u8; 32],
 }
 
-// From https://github.com/zcash-hackworks/zcash-test-vectors/blob/master/orchard_sinsemilla.py
+// From <https://github.com/zcash-hackworks/zcash-test-vectors/blob/master/orchard_sinsemilla.py>
```

Comment:
Optional: not needed.,
```suggestion
// From https://github.com/zcash-hackworks/zcash-test-vectors/blob/master/orchard_sinsemilla.py
```

Reply:
I fixed this one in [36fc845](https://github.com/ZcashFoundation/zebra/pull/4494/commits/36fc8453d4239291deb6e3f692a0d6b9253f17e8).